packages/cozy-pouch-link/examples/periodic-sync/index.js

```diff
@@ -77,57 +75,45 @@ class App extends React.Component {
         'state',
         JSON.stringify({
           url: newState.url,
-          token: newState.token && newState.token.toJSON()
+          token: newState.token
         })
       )
     })
   }
 
-  createManager() {
-    this.manager = new PouchManager([DOCTYPE], {
+  createClient() {
+    this.link = new PouchLink({
+      doctypes: ['io.cozy.bank.operations']
+    })
+    this.link.options.onSync = data => {
+      this.client.setData(data)
+    }
+    console.log(this.state.token)
```

Comment:
La ligne est peut etre à supprimer ?

Reply:
Oups, I did not meant to ship those commits